parsec/backend/swift_blockstore.py

```diff
@@ -33,9 +33,8 @@ def side_effect(key: str) -> str:
 
 def build_swift_slug(organization_id: OrganizationID, id: BlockID) -> str:
     # The slug uses the UUID canonical textual representation (eg.
-    # `CoolOrg/3b917792-35ac-409f-9af1-fe6de8d2b905`) where `BlockID.__str__`
-    # uses the short textual representation (eg. `3b91779235ac409f9af1fe6de8d2b905`)
-    return f"{organization_id.str}/{id.uuid}"
+    # `CoolOrg/3b917792-35ac-409f-9af1-fe6de8d2b905`)
+    return f"{organization_id.str}/{id}"
```

Comment:
Given that comment, we may want to be more explicit and provide a `as_hyphenated` method